docs/contributor/03-80-plan-updates.md

```diff
@@ -0,0 +1,30 @@
+# Service Plan Updates
+
+Kyma Environment Broker supports updating service plans. This feature allows you to change the plan of an existing Kyma runtime. However, only some plan changes are possible because the new plan must use the same provider. For example, you cannot switch from Amazon Web Services to Microsoft Azure.
```

Comment:
```suggestion
Kyma Environment Broker (KEB) supports updating service plans. This feature allows you to change the plan of an existing Kyma runtime. However, only some plan changes are possible because the new plan must use the same provider. For example, you cannot switch from Amazon Web Services to Microsoft Azure.
```